README.md

```diff
@@ -13,6 +13,19 @@ cd documentation
 npm run server
 ```
 
+## Documenting a component
+
+In the component's folder, create a `documentation` folder. Inside of that,
+The only requirement is an `index.md` file with a
+front-matter entry for `title`. Inside that file, describe the component.
+Render `html` examples by using the `example` Hugo shortcode:
+
+```
+{{< example fileName="example-1.html" >}}
+```
```

Comment:
Thoughts on showing a full example here, including the frontmatter and a description text?